test/extended/olm/olm.go

```diff
@@ -31,40 +35,40 @@ var _ = g.Describe("[sig-operator] OLM should", func() {
 	}{
 		{
 			fromAPIService: true,
-			group:          "packages." + operators,
+			group:          packagesGroupName,
 			version:        "v1",
 			plural:         "packagemanifests",
 		},
 		{
-			group:   operators,
+			group:   operatorsGroupName,
 			version: "v1",
 			plural:  "operatorgroups",
 		},
 		{
-			group:   operators,
+			group:   operatorsGroupName,
 			version: "v1alpha1",
 			plural:  "clusterserviceversions",
 		},
 		{
-			group:   operators,
+			group:   operatorsGroupName,
 			version: "v1alpha1",
 			plural:  "catalogsources",
 		},
 		{
-			group:   operators,
+			group:   operatorsGroupName,
 			version: "v1alpha1",
 			plural:  "installplans",
 		},
 		{
-			group:   operators,
+			group:   operatorsGroupName,
 			version: "v1alpha1",
 			plural:  "subscriptions",
 		},
 	}
 
 	for i := range providedAPIs {
 		api := providedAPIs[i]
-		g.It(fmt.Sprintf("be installed with %s at version %s", api.plural, api.version), func() {
+		g.It(fmt.Sprintf("be installed with %s at version %s [apigroup:%s]", api.plural, api.version, api.group), func() {
```

Comment:
I still wonder whether this will work. Though, running `openshift-tests` with --dry-run gives me:
```
"[sig-operator] OLM should be installed with catalogsources at version v1alpha1 [apigroup:operators.coreos.com] [Suite:openshift/conformance/parallel]"
"[sig-operator] OLM should be installed with clusterserviceversions at version v1alpha1 [apigroup:operators.coreos.com] [Suite:openshift/conformance/parallel]"
"[sig-operator] OLM should be installed with installplans at version v1alpha1 [apigroup:operators.coreos.com] [Suite:openshift/conformance/parallel]"
"[sig-operator] OLM should be installed with operatorgroups at version v1 [apigroup:operators.coreos.com] [Suite:openshift/conformance/parallel]"
"[sig-operator] OLM should be installed with packagemanifests at version v1 [apigroup:packages.operators.coreos.com] [Suite:openshift/conformance/parallel]"
"[sig-operator] OLM should be installed with subscriptions at version v1alpha1 [apigroup:operators.coreos.com] [Suite:openshift/conformance/parallel]"
```

Worth the try